js/main.js

```diff
@@ -1,5 +1,74 @@
+import {Player} from './player.js';
+import {Game} from './game.js';
 let play = document.querySelector('#start-game');
+let cells  = document.querySelectorAll('.cell');
+let label = document.querySelector('#player-name');
+let replay = document.querySelector('#replay');
+let formContainer = document.querySelector('#hidden');
+let winMessage = document.querySelector('#winner');
+let turn = true;
+let namePlayer1 = 'Player 1';
+let namePlayer2 = 'Player 2';
+let player1;
+let player2;
+let arrBoard = ['', '', '', '', '', '', '', '', ''];
+let game = Game(arrBoard);
+
+function addSymbol(cell,player ,symbol){
+    cell.innerHTML = `<${player}>${symbol}</${player}>`;
+}
+function toggleSymbols(toggle){
+    return toggle ? "X" : "O"
+}
```

Comment:
`Optional` Use the ES6 syntax to declare these two functions

Reply:
Can you show me the errors on your console? i can play, im not sure why is not working on you